src/webServiceClient.ts

```diff
@@ -0,0 +1,153 @@
+import https = require('https');
+import mmdb = require('maxmind');
+import * as models from './models';
+
+type servicePaths = 'city' | 'country' | 'insights';
```

Comment:
Should `servicePaths` be singular since only one of the values can be utilized?

Reply:
https://github.com/maxmind/GeoIP2-node/pull/21/commits/490eacfc38196cf0aa161e82a36afe57980a5d0c